harvard-stellenbosch-university.csl

```diff
@@ -31,26 +31,26 @@
       <if variable="editor">
         <names variable="editor">
           <name and="symbol" initialize-with="." delimiter=", "/>
-          <label form="short" prefix=", " text-case="capitalize-first"/>
+          <label form="short" prefix=" (" suffix=")"/>
         </names>
       </if>
     </choose>
     <choose>
       <if variable="translator">
         <names variable="translator">
           <label form="verb" text-case="capitalize-first" suffix=" "/>
-          <name and="symbol" delimiter=", "/>
+          <name and="symbol" delimiter=" (" suffix=")"/>
         </names>
       </if>
     </choose>
   </macro>
   <macro name="author">
     <names variable="author">
-      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never" et-al-min="9" et-al-use-first="8"/>
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never"/>
+      <et-al font-style="italic"/>
       <label form="short" prefix=" " text-case="capitalize-first"/>
       <substitute>
-        <names variable="editor"/>
-        <names variable="translator"/>
+        <text macro="editor-translator"/>
```

Comment:
what's the advantage of this? Using straight variables is generally preferred.

Reply:
`<names>` in substitute inherits the (already existing) label from the original `<names>` element, which you already have in l. 51, so you'd just have to adjust the label in l. 51. There is no author label, so the line is irrelevant for authors.

Reply:
exactly, yes, and that keeps the editor-translator more flexible, e.g.